log/simple-logger.go

```diff
@@ -0,0 +1,78 @@
+package log
```

Comment:
Up to this change I was OK with the abstraction.
Here is where I'm starting to get uncomfortable... IDK

Reply:
Yeah...
The thing is - that you need some default logger for when your user did not set a logger at all, right?